### DISCUSSION TITLE
think about it...

### DISCUSSION THREAD

**Discussion by idontveapc** [Docile-Alligator/Infinity-For-Reddit#1509] [General]
- #### Infinty's dev

They didn't give up, the app is open source and free(open) to anyone. Even reddit is trying eliminate other third party from their revenue, (__**`but infinity didn't try profit like other third parties showing ads or with subscription`**__), we still would be able to use infinity.

- #### Sync's dev(s)

They are now to get users money from another reddit alternative (lemmy). Even after taking a lot from them, for reddit, they're not able to pay reddit??

It means some just want to profit without paying.

> But they're not many like our dev.

**Comment by Magneticdud** [comment]
> they're not able to pay reddit??

Yes, at the exorbitant prices Reddit is charging (0.0024 per request), it's not possible to pay reddit, the 30% cut for Google and the rent of the proxy server that needs to authenticate each user request in order to check if the subscription is valid or it has been cracked/hacked/patched/revoked

By reddit own calculations, it costs $1 per month **in average including hundreds of thousands of inactive users**. Once you take out the thousands of free users that opened the app once a month and keep only the few paying subscribers that use the app daily for hours, the cost per user will raise to $15 per month, which is unaffordable
